### PR TITLE
Download files from a record

### PR DESCRIPTION
**Pull request by arshul**

### CODE REVIEW

File: ci/download.workflow [hunk 1]
```diff
@@ -0,0 +1,13 @@
+workflow "Download" {
+  resolves = "download"
+}
+action "download" {
+  uses = "./download"
+  secrets = ["ZENODO_API_TOKEN"]
```
Comment (ivotron) [inline ci/download.workflow:6]: do we need the token to download files? If not, then we can remove it from the `download` action

**Review by ivotron — CHANGES_REQUESTED** [review]
thanks @arshul! one more thing and we'll be done.

File: ci/download.workflow (hunk 1 above)
Comment (arshul) [reply ci/download.workflow:6]: When `ZENODO_FILES` value is not provided, in order to retrieve the list of files via the API I need the access token. [This](https://github.com/popperized/zenodo/blob/1de606ea43184dd216ea649d857f91e22a7ddf01/download/entrypoint.sh#L9) thing will not work without an access token.

**Review by ivotron — APPROVED** [review]

**Comment by ivotron** [comment]
thanks a lot @arshul !

**Pull request merged**